ext/cache/lib.deno_cache.d.ts

```diff
@@ -18,7 +29,12 @@ declare interface CacheStorage {
   delete(cacheName: string): Promise<boolean>;
 }
 
-/** @category Cache */
+/** 
+ * Provides a persistent storage mechanism for Request / Response object pairs 
+ * that are cached in long lived memory.
```

Comment:
It's not necessarily persistent (can be evicted). Also don't say in memory, as it could also be cached to disk